internal/api/grpc/user/v2/query.go

```diff
@@ -29,11 +29,11 @@ func (s *Server) GetUserByID(ctx context.Context, req *user.GetUserByIDRequest)
 }
 
 func (s *Server) ListUsers(ctx context.Context, req *user.ListUsersRequest) (*user.ListUsersResponse, error) {
-	queries, err := listUsersRequestToModel(req)
+	queries, filterOrgIds, err := listUsersRequestToModel(req)
 	if err != nil {
```

Comment:
Let's use the singular form, as we are only passing a single orgID.


```suggestion
	queries, filterOrgID, err := listUsersRequestToModel(req)
```

Reply:
This comment is not resolved